src/index.html

```diff
@@ -1,13 +1,430 @@
+              <p class="our-expertise__paragraph">We create additional value for companies,
+                products, services as well as verbal and visual ways to deliver it to the audience.
+              </p>
+            </div>
+
+            <div class="our-expertise__item">
+              <img
+              src="images/our-expertise/communication.png"
+              alt="Air Communication"
+              class="our-expertise__photo"
+              >
+
+              <h3 class="our-expertise__item-title">Communication</h3>
+              <p class="our-expertise__paragraph">We strive to create communications
+                that can increase media performance.
+                We use everything — words, meanings, stories, art, movies.
+              </p>
+            </div>
+
+            <div class="our-expertise__item">
+              <img
+              src="images/our-expertise/strategy.png"
+              alt="Air Strategy"
+              class="our-expertise__photo"
+              >
+
+              <h3 class="our-expertise__item-title">Strategy</h3>
+              <p class="our-expertise__paragraph">We create business growth strategies,
+                from the moment of its birth to the achievement of the necessary business indicators.
+              </p>
+            </div>
+          </article>
+        </section>
+
+        <section class="services" id="services">
+          <h2 class="services__title">Air is a full service creative agency</h2>
+
+          <article class="services__text">
+            <p class="services__paragraph">Deep analytics, strong strategy and bright creative ideas.</p>
+            <p class="services__paragraph">We are sure that first-rate job is possible only if all three components are united.
+            </p>
+          </article>
+
+          <article class="process">
+            <div class="process__card process__card--desktop">
+              <span class="process__number">001</span>
+              <ul class="process__items">
+                <li>Brand Development</li>
+                <li>Copywriting</li>
+                <li>Logo & Webite Design</li>
+                <li>Packaging</li>
+              </ul>
+
+              <span href="#our-expertise" class="process__button">
+                <div class="process__arrow"></div>
+                <p class="process__text">Learn more</p>
+              </span>
+            </div>
+
+            <div class="process__card process__card--transform">
+              <span class="process__number">002</span>
+              <ul class="process__items">
+                <li>Content Production</li>
+                <li>Graphic Design</li>
+                <li>Video Production</li>
+                <li>Post Production</li>
+              </ul>
+
+              <span href="#our-expertise" class="process__button">
+                <div class="process__arrow"></div>
+                <p class="process__text">Learn more</p>
```

Comment:
_The "Learn more" button should lead to the closest block (Our expertise)_ - point from the cheklist